libs/pilight/protocols/433.92/alecto_ws1700.c

```diff
@@ -75,13 +75,17 @@ static void parseCode(void) {
 		}
 	}
 
-	id = binToDecRev(binary, 0, 11);
+	header = binToDecRev(binary, 0, 3);
+	if (header != 5) {
+		return;
+	}
+	id = binToDecRev(binary, 4, 11);
 	battery = binary[12];
-	temperature = ((double)binToDecRev(binary, 18, 27));
+	temperature = (double)binToDecRev(binary, 16, 27);
 	humidity = (double)binToDecRev(binary, 28, 35);
 
-	if(temperature > 511) {
-		temperature -= 1023;
+	if (binary[16]) {
+	    temperature = temperature - 4096;
```

Comment:
Are you using Github to edit the code, or are you using git?

If you're using git you can make a new commit, and the follow these instructions to "squash" this current commit with the new one for the -= operator:
http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit

Then you run
```git push origin HEAD -f```
to force push your change to the same branch. GitHub will automatically update the pull request.

Whatever you do, just please don't close this PR and open a new one :wink: